(node:1663591) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-session-41505',
  containerIp: '10.0.3.63',
  mitmPort: 41505,
  autoAccessFixed: 1
}
2024-11-07 11:29:05.523 - [Info] MITM Version: 2
2024-11-07 11:29:05.525 - [Info] Auto Access Enabled: true
2024-11-07 11:29:05.525 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-07 11:29:05.544 - [Info] SSH man-in-the-middle server for 10.0.3.63 listening on 10.0.3.1:41505
2024-11-07 11:31:38.120 - [Debug] [Connection] Attacker connected: 8.219.239.191 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 11:31:39.597 - [Debug] [Auth] Attacker 8.219.239.191 trying to authenticate with 'password'
2024-11-07 11:31:39.598 - [Debug] [Auto Access] Attacker: 8.219.239.191, Threshold: 1, Attempts: 1
2024-11-07 11:31:39.599 - [Debug] [Auto Access] Compromising the honeypot
2024-11-07 11:31:39.599 - [Debug] [Auto Access] Adding the following credentials: 'admin:Qazxsw123'
2024-11-07 11:31:39.871 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-07 11:31:39.873 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-07 11:31:39.874 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.63
2024-11-07 11:31:39.967 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-07 11:31:40.952 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-07 11:31:42.512 - [Debug] [Connection] Attacker connected: 8.219.239.191 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 11:31:44.025 - [Debug] [Auth] Attacker 8.219.239.191 trying to authenticate with 'password'
2024-11-07 11:31:44.026 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-07 11:31:44.027 - [Debug] [Auth] Attacker: 8.219.239.191 has so far made 1 attempts. Remaining: 5 attempts
2024-11-07 11:31:44.517 - [Debug] [Connection] Attacker closed the connection
2024-11-07 11:31:44.755 - [Debug] [Connection] Attacker connected: 8.219.239.191 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
